site/content/docs/5.0/examples/sign-in/index.html

```diff
@@ -11,10 +11,16 @@
   <form>
     <img class="mb-4" src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" alt="" width="50" height="50">
     <h1 class="h3 mb-3 fw-normal">Please sign in</h1>
-    <label for="inputEmail" class="visually-hidden">Email address</label>
-    <input type="email" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
-    <label for="inputPassword" class="visually-hidden">Password</label>
-    <input type="password" id="inputPassword" class="form-control" placeholder="Password" required>
+
```

Comment:
need to review the form display

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380